plugins/thunderbolt/meson.build

```diff
@@ -1,8 +1,13 @@
 cargs = ['-DG_LOG_DOMAIN="FuPluginThunderbolt"']
 
+fu_plugin_thunderbolt_sources = [
+  'fu-plugin-thunderbolt.c',
+  'fu-thunderbolt-image-validation.c',
+]
+
 shared_module('fu_plugin_thunderbolt',
   sources : [
-    'fu-plugin-thunderbolt.c',
+    fu_plugin_thunderbolt_sources,
```

Comment:
You can just include `fu-thunderbolt-image-validation.c` here both times, it's only one file.